src/features/pages/partners/deriv-prime/why-deriv-prime/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import * as styles from './why-deriv-prime.module.scss'
+import { card_data } from './data'
+import Typography from 'features/components/atoms/typography'
+import FlexBox from 'features/components/atoms/flex-box'
+import { Localize, WithIntl } from 'components/localization'
+
+const WhyDerivPrime = () => {
+    return (
+        <FlexBox.Box
+            container={'fluid'}
+            direction={'col'}
+            padding_inline={'4x'}
+            pt={'20x'}
+            md={{ padding_inline: '0x', padding_block: '40x' }}
+        >
+            <FlexBox.Box
+                direction={'col'}
+                margin_block={'40x'}
+                className={styles.container}
+                md={{ justify: 'center', margin_block: '40x' }}
+            >
+                <Typography.Heading align={'center'} className={styles.header}>
+                    <Localize translate_text={'_t_Why choose Deriv Prime?_t_'} />
+                </Typography.Heading>
+                <FlexBox.Box
+                    direction={'col'}
+                    align={'center'}
+                    wrap="wrap"
+                    md={{ direction: 'row', justify: 'center', mt: '20x' }}
+                >
+                    {card_data.map((card) => {
+                        return (
+                            <FlexBox.Box
+                                key={card.title}
+                                className={styles.card}
+                                direction={'col'}
+                                justify={'start'}
+                                align={'start'}
+                                radius={'8x'}
+                                padding={'8x'}
+                                mt={'12x'}
+                                md={{ padding: '12x', mr: '12x', mt: '0x' }}
```

Comment:
`gap` property to parent would be good to make gap between cards instead of `mr` and `mt`